### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by morris** [morris/vanilla-todo#12]
During the initial study, not much was invested into accessibility concerns. I believe for a more complete result, accessibility should be taken into account. Looking for feedback and ideas how to approach this:

- How accessible is the application right now?
- What are the accessibility gaps?
- How could we close these gaps?

PRs are also highly welcome!

**Comment by harrisonJones12** [comment]
Hi Morris, I apologize for getting back to you so late on the last issue. However this looks like something I would like to investigate and fix :D the potential accessibility issues present.
